src/Speckle_filter/_helper_functions.jl

```diff
@@ -0,0 +1,44 @@
+
+""""
+Helper functions
+Used, e.g., in filter comparisions.
+"""
+
+function variance_to_mean_ratio(image_window)
+    # ϵ added for numerical stability. #Simon, is this an issue in julia?
+    ϵ = 0.00000001
+    return (std(image_window) / (mean(image_window)^2 + ϵ))
```

Comment:
I guess it is still needed, but you can do eps() = smallest machine precision:
`return (std(image_window) / (mean(image_window)^2 + eps()))`